molgenis-dataexplorer/src/main/java/org/molgenis/dataexplorer/controller/DataExplorerController.java

```diff
@@ -182,9 +183,20 @@ else if (moduleId.equals("annotators"))
 			model.addAttribute("annotationRun", annotationRun);
 			model.addAttribute("entityName", entityName);
 		}
+
 		return "view-dataexplorer-mod-" + moduleId; // TODO bad request in case of invalid module id
 	}
 
+	@RequestMapping(value = "/copy", method = GET)
+	@ResponseBody
+	public boolean showCopy(@RequestParam("entity") String entityName)
+	{
+		boolean showCopy = false;
+		showCopy = molgenisPermissionService.hasPermissionOnEntity(entityName, Permission.READ)
+				&& dataService.getCapabilities(entityName).contains(RepositoryCapability.WRITABLE);
```

Comment:
Yes, but I mean when the repository isn't writable and the user has count permission, I think he should be able to copy it to a new repo that's created in the default repositorycollection. No?